.github/workflows/CI.yaml

```diff
@@ -88,12 +88,17 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Install package
+    - uses: conda-incubator/setup-miniconda@v2
```

Comment:
conda comes with a lot of pre-installed stuff, even at the miniconda level. I would instead use setup-python which gives you a bit more of a barebones experience: https://github.com/actions/setup-python

Reply:
yep will fix.